src/Apps/Search/Components/NavigationTabs.tsx

```diff
@@ -120,7 +124,10 @@ export class NavigationTabs extends React.Component<Props> {
   render() {
     return (
       <Flex mx={[-2, 0]}>
-        <TabCarousel tabs={this.tabs()} />
+        <TabCarousel
+          key={`tab-carousel-${this.props.term}`}
```

Comment:
Setting a key, ensure that flickity doesn't freak out react when unmounting page. (Only happens occasionally)